packages/image_picker/image_picker/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -389,6 +449,14 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
+		6801C8322555D726009DAF8D /* Sources */ = {
```

Comment:
This is the Sources build phase that says which files should be compiled into the xctest bundle.  So `ImagePickerFromGalleryUITests` in this case.